go.mod

```diff
@@ -131,6 +131,7 @@ require (
 	k8s.io/kubelet v0.22.13
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 	sigs.k8s.io/controller-runtime v0.11.2
+	sigs.k8s.io/controller-tools v0.7.0
```

Comment:
We seem to be using `v0.8.0` for the main operator.
Just curious why you chose `v0.7.0` here? Just to make it clear: it's absolutely fine if there was an actual reason. Otherwise it might make sense to be consistent.

Reply:
+1 I thought it uses newer k8s deps but I was checking v0.9 not v0.8. I'll fix it